association_measures/measures.py

```diff
@@ -177,11 +172,11 @@ def calculate_measures(df, measures=None):
         if isinstance(measures[0], str):
             measures = [ams_all[k] for k in measures if k in ams_all.keys()]
     else:
-        measures = [ams_all[k] for k in ams_all if k != 'hypergeometric_likelihood']
```

Comment:
What's happening here?

Reply:
When giving a list of `measures` to `calculate_measures`, the function translates those in the actual measures:
`for measure in measures: df[measure.__name__] = measure(df)`
The code before just determines what measures can be selected. Current implementation disables `hypergeometric_likelihood`.

Reply:
this seems a bit opaque, I'll have a look at it